kitsune/wiki/jinja2/wiki/edit.html

```diff
@@ -19,83 +20,59 @@
 
   <div class="sumo-page-section">
     <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
-      <h1 class="sumo-page-heading">{{ _('<em>Editing</em> {title}')|fe(title=document.title) }}</h1>
+      <h1 class="sumo-page-heading">{{ _('<em>Editing Content For:</em><br>{title}')|fe(title=document.title) }}</h1>
       {% if document.needs_change %}
         <p>
           {% trans comment=document.needs_change_comment %}
             <label>Needs Change:</label> {{comment}}
           {% endtrans %}
         </p>
       {% endif %}
-
-      {% if document_form %}
-        <div id="document-form">
-          <details {% if disclose_description %} open="open"{% endif %}>
-            <summary class="sumo-page-subheading">{{ _('Edit Description') }}</summary>
-            {{ errorlist(document_form) }}
-            <form action="" method="post" data-json-url="{{ url('wiki.json') }}" data-document-id="{{ document.id }}">
-              {% csrf_token %}
-                {% for field in document_form.visible_fields() if
-                   (field.name != 'is_localizable' or not document.translations.exists()) %}
-                  <div class="field {% if field.name == 'needs_change_comment' %}has-large-textarea comment{% endif %}">
-                    {{ field|label_with_help }}{{ field }}
+            {% if request.user.is_staff %}
```

Comment:
why are we introducing this change?

Reply:
When I spoke to Abby about this effort, she asked that only staff be allowed to edit the Document Meta (title, slug, etc).

Reply:
I wasn't enforcing in the view - I'll look at that today.